Feature/add loading spinner and message if no data

jhommark opened the pull request.

tma-isbx commented on client/app/dashboard/model/list/ModelList.js:
```diff
@@ -468,39 +478,39 @@ angular.module('dashboard.Dashboard.Model.List', [
-          }
-          */
-          
+           if ($scope.model && $scope.model.options.relations) {
```
Whitespace clean up broke the formatting of this comment.

tma-isbx commented:
Please resolve conflicts, thanks!

tma-isbx commented:
@jhommark Looks good 👍

@coolwr Should be ready for your review and merge after @jhommark resolves conflicts. Much of the diff is due to whitespace clean up. The actual relevant changes are fairly short.

jhommark commented:
Hi @tma-isbx and @coolwr,

I updated the branch to remove the whitespaces cleanup that happened and adjusted by my IDE. I patched it with my new local branch to put only the relevant changes.

tma-isbx commented:
@coolwr Ready for your review and merge.
